### PR TITLE
Add podge to opam

### PR DESCRIPTION
**Pull request by fxfactorial**

### CODE REVIEW

File: packages/podge/podge.0.2/descr [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Shortcuts and helpers for common tasks in OCaml ecosystem
+
+If you're doing any modern OCaml then you're doubtlessly annoyed by
+the state of libraries and committing to one of the big ones can be
+restricting. Podge is a single modules containing specialized modules
```
Comment (avsm) [inline packages/podge/podge.0.2/descr]: minor typo: should be "Podge is a single module"

File: packages/podge/podge.0.2/descr [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+for their respectives usages for seemginly common tasks.
```
Comment (avsm) [inline packages/podge/podge.0.2/descr]: s/seemginly/seemingly

File: packages/podge/podge.0.2/descr (hunk 1 above)
Comment (fxfactorial) [inline packages/podge/podge.0.2/descr]: Oh doh. What is the easiest way to fix these embarrassing mistakes?

Comment (domsj) [inline packages/podge/podge.0.2/descr]: With git you can amend commits and then force push the branch, that would've been easiest in your case

**Pull request closed without merging**